tiatoolbox/tools/patchextraction.py

```diff
@@ -545,7 +618,7 @@ def get_patch_extractor(method_name, **kwargs):
 
     Args:
         method_name (str): name of patch extraction method, must be one of "point" or
-          "slidingwindow".
+            "slidingwindow".
```

Comment:
Add "These are case insensitive".

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/281/commits/d067b29e6e28646aa6552de10919f37690f32c3e